python.md

```diff
@@ -125,22 +125,41 @@ to the number of threads to use, the default is to use a single thread.
 
 ## 3. Other common Python packages
 
+### 3.1 cffi
 
-### 3.1 Pillow
-
-On **AmazonLinux2 and RedHat**:
+On **AmazonLinux2/Redhat**:
 
 ```
-sudo yum install libtiff-devel libjpeg-devel openjpeg2-devel zlib-devel freetype-devel lcms2-devel libwebp-devel tcl-devel tk-devel harfbuzz-devel fribidi-devel libraqm-devel libimagequant-devel libxcb-devel
-pip3 install --user pillow
+sudo yum install "@Development tools" libffi-devel
+pip3 install --user cffi
 ```
 
 On **Ubuntu**:
+
 ```
-sudo apt-get install libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python3-tk libharfbuzz-dev libfribidi-dev libxcb1-dev
-pip3 install --user pillow
+sudo apt-get install libffi-dev build-essential
+pip3 install --user cffi
 ```
 
+### 3.2 Pillow
+
+As of June 30, 2020, Pillow 7.2.0 was released as a binary wheel for Arm platforms, but contains a bug that prevents it from loading
+properly on distributions that use 64kB pages and must be compiled from source, i.e. Centos8 and Redhat8.  To determine if you need to
+install Pillow from source, do the following:
+
+On **RedHat**:
+
+```bash
+if [[ $(getconf PAGESIZE) > 4096 ]]; then
+    sudo yum install libtiff-devel libjpeg-devel openjpeg2-devel zlib-devel freetype-devel lcms2-devel libwebp-devel tcl-devel tk-devel harfbuzz-devel fribidi-devel libraqm-devel libimagequant-devel libxcb-devel
+    pip3 install --user --no-binary pillow
+else
+    pip3 install --user pillow
+fi
+```
+
```

Comment:
its not clear that ubuntu/amazonlinux doesnt need any of this.     so i would put "On **Ubuntu and AmazonLinux2**: 
 just follow normally install pip3 --user pillow"

Reply:
Done.